### ISSUE TITLE
compilation fail on armv7 (raspberry pi 3 B, raspbian)

### ISSUE THREAD

**Issue by Tapmemer** [yvt/openspades#512]
1%] Built target OpenSpades_Resources
[  1%] Built target OpenSpades_Unix
[  5%] Built target Angelscript_addons
[ 17%] Built target Angelscript
[ 17%] Linking CXX executable ../bin/openspades
../lib/libAngelscript.a(as_callfunc.cpp.o): In function `CallSystemFunction(int, asCContext*)':
as_callfunc.cpp:(.text+0x2c4): undefined reference to `CallSystemFunctionNative(asCContext*, asCScriptFunction*, void*, unsigned long*, void*, unsigned long long&, void*)'
collect2: error: ld returned 1 exit status
Sources/CMakeFiles/OpenSpades.dir/build.make:5748: recipe for target 'bin/openspades' failed
make[2]: *** [bin/openspades] Error 1
CMakeFiles/Makefile2:224: recipe for target 'Sources/CMakeFiles/OpenSpades.dir/all' failed
make[1]: *** [Sources/CMakeFiles/OpenSpades.dir/all] Error 2
Makefile:149: recipe for target 'all' failed
make: *** [all] Error 2

do i need some extra build parameters or edit the makefile?

i have all dependecies and the extra dependencies (for if the requirements bug happens)

**Comment by noway421** [comment]
Someone tried compiling and compiled it I think, hopefully they could shed some light on how it can be done. Also would be good to add it to wiki

**Comment by Tapmemer** [comment]
i think the limking CXX executable is not possible due to raspbian. is there any way to remove or disable the command linking in the makefile/sourcecode?

**Comment by feikname** [comment]
Related: #416

**Comment by Tapmemer** [comment]
he compiled it on armv6 probably a raspberry pi or pi 2. armv6 and a raspberry pi 3B are different from eachoter

**Comment by Tapmemer** [comment]
im trying to compile it again with the new CMakeLists.txt from jextrevor. i hope it works

**Comment by Tapmemer** [comment]
nope. i cannot compile it with the cmakelists.txt. i changed it so it would support armv7vl but no

**Comment by feikname** [comment]
I will close this one and leave the dicussion at #416

**Comment by feikname** [comment]
> he compiled it on armv6 probably a raspberry pi or pi 2. armv6 and a raspberry pi 3B are different from eachoter

@Tapmemer I am deeply sorry, I totally missed your comment. Although related, those issues should probably be kept apart.
